### ISSUE TITLE
✨ Feedback: Better UX: Error: This file/folder doesn't exist.

### ISSUE THREAD

**Issue by dawidmachon** [samuel-lucas6/Kryptor#18]
### Description

After using the "-e -p" options command, the program first asks you for the password and then informs you that you typed the wrong file name. It would be more logical to reverse the check.

```
X:\Some\Windows\Path\kryptor-windows> ./kryptor -e -p MyMissTypoFile.txt
Enter a password (leave empty for a random passphrase):
Retype password:

MyMissTypoFile.txt - Error: This file/folder doesn't exist.
```

**Comment by samuel-lucas6** [comment]
I completely agree, but sadly it's not quite that simple. Because Kryptor supports multiple files and/or folders being specified at once, each path is being checked in a loop to process each file/folder one at a time, and there are also several checks (e.g. whether the file has the Kryptor magic bytes).

I think the reason I did it this way was to continue processing existing files/folders rather than giving an error if only one path is wrong. I will take a proper look when I get the time. There's got to be a better way of writing it.

**Comment by samuel-lucas6** [comment]
I promise I will get around to looking at this, but I have exams throughout June unfortunately, so I haven't been doing much programming lately.

**Comment by samuel-lucas6** [comment]
Fixed in [v3.0.3](https://github.com/samuel-lucas6/Kryptor/releases/tag/v3.0.3).
